docs/api/Auth0.Core.Exceptions.RateLimit.html

```diff
@@ -223,7 +225,8 @@ <h5 class="returns">Returns</h5>
     <tbody>
       <tr>
         <td><a class="xref" href="Auth0.Core.Exceptions.RateLimit.html">RateLimit</a></td>
-        <td></td>
+        <td><p>Instance of <a class="xref" href="Auth0.Core.Exceptions.RateLimit.html">RateLimit</a> containing parsed rate limit headers.</p>
+</td>
```

Comment:
Docs wasn't generated when introducing this changes https://github.com/auth0/auth0.net/commit/58dc202df1926d90d8b8bbfab050df63cf97e4ec